projects/admin.py

```diff
@@ -30,13 +29,11 @@ class PublicationAdmin(ProjectFields, admin.ModelAdmin):
         "doi",
         "link",
         "added_by_username",
-        "entry_created_date",
         "status",
-        "approved_with",
```

Comment:
I see no migration that converts this data to the charfield. Is there any data in this column or was it new (and so doesn't exist in prod)?

Reply:
Yes. I should also have a migration/workflow to store the existing data in new model